app/lib/forms/your_employer.rb

```diff
@@ -0,0 +1,30 @@
+module Forms
+  class YourEmployer < Base
+    QUESTION_NAME = :employer_name
+
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
+
+    def self.permitted_params
+      [QUESTION_NAME]
+    end
+
+    def question
+      OpenStruct.new(
+        type: :text_field,
+        name: QUESTION_NAME,
+        label: I18n.t("registration_wizard.your_employer.label"),
+        hint: I18n.t("registration_wizard.your_employer.hint"),
```

Comment:
The form builder does have [localisation built in](https://govuk-form-builder.netlify.app/building-blocks/localisation/). You could remove these two lines and restructure the locales a little to get the same outcome:

```patch
diff --git a/config/locales/en.yml b/config/locales/en.yml
index b75987a..dd731c9 100644
--- a/config/locales/en.yml
+++ b/config/locales/en.yml
@@ -197,6 +197,10 @@ en:
       label: "What is your role?"
       hint: "For example: Administrator, business manager"

-    your_employer:
-      label: "What organisation are you employed by?"
-      hint: "For example 'Essex County Council', or 'I'm self employed'"
+  helpers:
+    label:
+      registration_wizard:
+        employer_name: What organisation are you employed by?
+    hint:
+      registration_wizard:
+        employer_name: "For example 'Essex County Council', or 'I'm self employed'"
```

Totally your call, the option's there though.

Reply:
I've swapped this over, it's a far nicer solution than hooking this up manually.